Src/WitsmlExplorer.Api/Extensions/CollectionExtensions.cs

```diff
@@ -6,7 +6,7 @@ namespace WitsmlExplorer.Api.Extensions
 {
     public static class CollectionExtensions
     {
-        public static IEnumerable<IEnumerable<T>> Chunk<T>(this IEnumerable<T> data, int chunkSize)
+        public static IEnumerable<IEnumerable<T>> Chunked<T>(this IEnumerable<T> data, int chunkSize)
```

Comment:
This class/extension method can now be removed since it is included in C# 10:

https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.chunk?view=net-6.0

Reply:
Nice :) I did not realize that it was the exact same implementation. I will remove the extension